dbt_platform_helper/domain/maintenance_page.py

```diff
@@ -214,12 +216,13 @@ def add_maintenance_page(
                     "BypassIpFilter",
                     next(rule_priority),
                     service_conditions,
+                    [{"Key": "service", "Value": svc.name}],
                 )
 
                 # add to accumilating list of conditions for maintenace page rule
                 maintenance_page_host_header_conditions.extend(service_conditions)
 
-            self.io.info(
+            self.io.debug(
```

Comment:
Why has this changed to debug?

Reply:
I think this one should be info, good catch. I think other logs should be debug as we may want to suppress them for normal logging but still have a way to enable debugging logs with a flag